app/Metadata/Versions/GitHubVersion.php

```diff
@@ -72,7 +72,7 @@ public function isRelease(): bool
 	 */
 	public function isMasterBranch(): bool
 	{
-		return $this->remote instanceof GitCommits && $this->localBranch === self::MASTER;
+		return $this->remote instanceof GitTags || ($this->remote instanceof GitCommits && $this->localBranch === self::MASTER);
```

Comment:
With this new condition is the name of the method still pertinent ?

Reply:
Yes, because tags are only on master. :)